### PR TITLE
Mute the audio when you are in meetings.

### PR DESCRIPTION
**Pull request by vamsiv96**
* Allows an app to mute the incoming audio speaker in the meeting. Unmute is not allowed
   * @param callback Callback contains 2 parameters, error and result.
   * error can either contain an error of type SdkError, incase of an error, or null when toggle is successful
   * result can either contain the true/false value, incase of a successful toggle or null when the toggling fails
   * result: True means incoming audio is muted

### CODE REVIEW

File: src/public/meeting.ts [hunk 1]
```diff
@@ -159,6 +159,21 @@ export namespace meeting {
...
   }
 
+  /**
+   * Allows an app to mute the incoming audio speaker in the meeting. Unmute is not allowed
+   * @param callback Callback contains 2 parameters, error and result.
+   * error can either contain an error of type SdkError, incase of an error, or null when toggle is successful
+   * result can either contain the true/false value, incase of a successful toggle or null when the toggling fails
+   * result: True means incoming audio is muted
+   */
+  export function muteViaMeshApp(callback: (error: SdkError | null, result: boolean | null) => void): void {
```
Comment (ronakbakshi) [inline src/public/meeting.ts:169]: Update the name of the API. This API name is limiting to Mesh app only, but our API should be open to all apps. I suggest something like `muteMicrophone`

File: src/public/meeting.ts [hunk 2]
```diff
@@ -159,6 +159,21 @@ export namespace meeting {
...
+    if (!callback) {
+      throw new Error('[mute via mesh app] Callback cannot be null');
+    }
+    ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
```
Comment (ronakbakshi) [inline src/public/meeting.ts:173]: start a discussion with Me, Kumud, Lisa, Ajay and check for which contexts we should support this API?

**Comment by MarcinJuraszek** [comment]
Did this go through SDK review process already?

File: src/public/meeting.ts [hunk 3]
```diff
@@ -159,6 +159,20 @@ export namespace meeting {
     sendMessageToParent('toggleIncomingClientAudio', callback);
   }
 
+  /**
+   * Allows an app to mute the microphone in the meeting.
+   * @param callback Callback contains 2 parameters, error and result.
+   * error can either contain an error of type SdkError, incase of an error, or true when mute is successful
```
Comment (ronakbakshi) [inline src/public/meeting.ts]: * error can either contain an error of type SdkError, incase of an error, or null when mute is successful
   * result can either contain the true value, incase of a successful mute or null when the mute fails
   * result: True means microphone is muted

**Comment by TrevorJoelHarris** [comment]
The author didn't answer @MarcinJuraszek 's question about design review and hasn't updated this PR in almost 6 months. I am going to close it, please feel free to re-open the PR once you've gone through a design review for this change. Let me know if you need help going through that process.

**Pull request closed without merging**